packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/attributeStartingAt..st

```diff
@@ -0,0 +1,10 @@
+attributes
+attributeStartingAt: openingMarkdownDelimiterEmphasis
+
+	| attributeDelimiterLength start end attributeClass |
+
+	attributeDelimiterLength := (self attributeDelimiterLengthWith: openingMarkdownDelimiterEmphasis).
```

Comment:
We actually discussed this name a bit. What we are determining is the length of the part of the delimiter that belongs to the attribute (it's either 1 or 2). We thought about having `Part` in the name swell but we thought it would be too long. Thats why we settled for `attributeDelimiterLength`.

Reply:
We could rename it to `delimiterLength` as soon as we rename `MarkdownEmphasisDelimiter` to `MarkdownEmphasisDelimiterRun`. But we want to discuss this renaming with everyone